app-modules/service-management/tests/Jobs/ServiceMonitoringCheckJobTest.php

```diff
@@ -95,3 +96,31 @@
             fn () => ServiceMonitoringFrequency::TwentyFourHours,
         ]
     );
+
+it('Handle unresolvable host errors gracefully', function ($frequency) {
+    Http::fake(function () {
+        throw new ConnectionException('Connection failed');
```

Comment:
That is not what the execption message looks like. Look at the error in the ticket.